apps/opik-backend/src/main/java/com/comet/opik/domain/TraceDAO.java

```diff
@@ -232,6 +235,16 @@ AND id in (
             ;
             """;
 
+    private static final String TRACE_CNT_BY_WORKSPACE_ID = """
+                SELECT
+                     workspace_id,
+                     COUNT(1) as traces_count
```

Comment:
For a more precise count, you should probably count the number of distinct trace id per workspace:
-`COUNT(DISTINCT id)`
The reason is that for `ReplacingMergeTree` engine, like in trace table, there might transiently be multiple records for the same record that are collapsed into one. 

This is to handle the update flow. We're going to get rid of the update flow, but until that, there's a possibility of that happening (every trace is inserted twice always).

Reply:
Done, thank you, good catch